### ISSUE TITLE
Pretty email template fails accessibility contrast test

### ISSUE THREAD

**Issue by berthelemy** [willianmano/moodle-theme_moove#294]
Hi there,

In https://github.com/willianmano/moodle-theme_moove/blob/master/templates/mod_forum/forum_post_email_htmlemail_body.mustache

The message body has a background color: #FFF and a foreground color: #999.

This looks great. Unfortunately it produces a contrast ratio of 2.84:1 which fails accessibility tests.

I know I can edit the code, but I was hoping we could make the change centrally so I don't have to change it every time I update.

Can we go for #555 instead for the foreground colour, as that gives a contrast ratio which passes both AA and AAA tests?

Thanks,

Mark

**Comment by willianmano** [comment]
@berthelemy thank you very much, I am always concerned about accessibility issues.
